docs/docs/connectors/your-own-website.mdx

```diff
@@ -116,6 +116,13 @@ socketio:
 The first two configuration values define the event names used by Rasa Open Source
 when sending or receiving messages over socket.io.
 
+Metadata can be added to the message by adding an object named `metadata` to the message. The `metadata_key` setting can be used to supply metadata on another object name. The default value for `metadata_key` is `metadata`. For example:
```

Comment:
Another clarifying question - sorry, looking at this with fresh eyes after a while 😄 - What does `another object name` refer to in this sentence "The `metadata_key` setting can be used to supply metadata on another object name."

Reply:
Sounds good 👍🏻

Reply:
pushed updated text and resolved conflict